tt_metal/llrt/blackhole/bh_hal_active_eth.cpp

```diff
@@ -58,10 +58,14 @@ HalCoreInfoType create_active_eth_mem_map() {
     mem_map_sizes[utils::underlying_type<HalL1MemAddrType>(HalL1MemAddrType::LAUNCH_MSG_BUFFER_RD_PTR)] = sizeof(uint32_t);
     mem_map_sizes[utils::underlying_type<HalL1MemAddrType>(HalL1MemAddrType::FW_VERSION_ADDR)] = sizeof(std::uint32_t);
 
-    // TODO (abhullar): This should be NumEthDispatchClasses
-    std::vector<std::vector<uint8_t>> processor_classes(1);
-    std::vector<uint8_t> processor_types{0};
+    std::vector<std::vector<HalJitBuildConfig>> processor_classes(NumEthDispatchClasses - 1);
```

Comment:
I went with the approach of making it look like BH active eth only has one processor. This means when we create active eth kernels the `processor` part of the EthernetConfig will be ignored (similar to being ignored on WH)